### PR TITLE
Add logo to README [SAME VERSION]

### PR DESCRIPTION
**Pull request by kate-goldenring**
**What this PR does / why we need it**:
We have a logo! Let's show it off.

**Special notes for your reviewer**:
Also, removed some image files that appear to be duplicates.

### CODE REVIEW

**Comment by kate-goldenring** [comment]
@flynnduism Are the images I removed here indeed duplicates/ old versions?

File: README.md [hunk 1]
```diff
@@ -10,10 +10,16 @@
 [![Build Agent](https://github.com/deislabs/akri/workflows/Build%20Agent/badge.svg?branch=main&event=push)](https://github.com/deislabs/akri/actions?query=workflow%3A%22Build+Agent%22)
 [![Test K3s, Kubernetes, and MicroK8s](https://github.com/deislabs/akri/workflows/Test%20K3s,%20Kubernetes,%20and%20MicroK8s/badge.svg?branch=main&event=push)](https://github.com/deislabs/akri/actions?query=workflow%3A%22Test+K3s%2C+Kubernetes%2C+and+MicroK8s%22)
 
+<img src="art/icon/akri-icon-light.svg" alt="Akri logo" width="100">
```
Comment (kate-goldenring) [inline README.md:13]: would be great to conditionally use the dark/light logo depending on whether people are using dark/light mode. Not sure how to do this, though.

Comment (bfjelds) [reply README.md:13]: just an idea, but, what about using: akri-logo-horizontal-light.svg instead?  you've done what [K8s](https://github.com/kubernetes/kubernetes/blob/master/README.md) did, but i kind of like what [etcd](https://github.com/etcd-io/etcd/blob/master/README.md) did.

Comment (kate-goldenring) [reply README.md:13]: I was trying to decide between those options, too. I didnt like how [thanos](https://github.com/thanos-io/thanos) removed the title, worring about if the image didn't render, but etcd doing both seems like a good balance. I'm open to changing it

Comment (bfjelds) [reply README.md:13]: i feel like the cube icon just kind of dangles where it is.  i'm not immediately connecting it as the logo.

Comment (kate-goldenring) [reply README.md]: Made the logo and wordmark the title. I think it looks much better. Thoughts?

**Comment by kate-goldenring** [comment]
@edrickwong what do you think?

File: README.md (hunk 1 above)
Comment (bfjelds) [reply README.md]: i like it.  my only nit is i wish it were a little smaller.  lol.

**Review by bfjelds — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (romoh) [reply README.md]: New updates looks good. +1 on being a bit smaller.

**Comment by edrickwong** [comment]
Smaller logo looks good to me. Hoping GitHub will support light/dark mode in readmes soon.

**Pull request merged**